code/game/machinery/kitchen/cooking_machines/_appliance.dm

```diff
@@ -214,6 +207,8 @@
 	else if(istype(check, /obj/item/weapon/disk/nuclear))
 		to_chat(user, "<span class='warning'>You can't cook that.</span>")
 		return 0
+	else if(istype(I, /obj/item/weapon/crowbar) || istype(I, /obj/item/weapon/screwdriver) || istype(I, /obj/item/weapon/storage/part_replacer))
+		return 0
```

Comment:
```suggestion
		return FALSE
```
For all of these, if it's meant to be false/true instead of a number.

Reply:
Done